include/mapbox/geojsonvt/clip.hpp

```diff
@@ -59,7 +59,7 @@ class clipper {
         for (const auto& ring : polygon) {
             const auto new_ring = clipRing(ring);
```

Comment:
the const qualifier needs to be removed for std::move

Reply:
fixed, and moved temp ring in multi polygon converter